pkg/grpc/exec/exec.go

```diff
@@ -166,202 +175,148 @@ type MsgCloneEventUnix struct {
 	processapi.MsgCloneEvent
 }
 
+var _ notify.CacheRetry = (*MsgCloneEventUnix)(nil)
+
 func (msg *MsgCloneEventUnix) Notify() bool {
 	return false
 }
 
-func (msg *MsgCloneEventUnix) RetryInternal(ev notify.Event, timestamp uint64) (*process.ProcessInternal, error) {
-	return nil, process.AddCloneEvent(&msg.MsgCloneEvent)
-}
+func (msg *MsgCloneEventUnix) Retry(ev notify.Event, ca *notify.CacheActions, pid uint32, timestamp uint64) error {
```

Comment:
Can we add a comment on why this needs its own retry function? The same for the other events with a custom function.